homeassistant/components/dyson/air_quality.py

```diff
@@ -103,44 +91,42 @@ def particulate_matter_2_5(self):
         """Return the particulate matter 2.5 level."""
         if self._device.environmental_state:
             return int(self._device.environmental_state.particulate_matter_25)
+        return None
 
     @property
     def particulate_matter_10(self):
         """Return the particulate matter 10 level."""
         if self._device.environmental_state:
             return int(self._device.environmental_state.particulate_matter_10)
+        return None
 
     @property
     def nitrogen_dioxide(self):
         """Return the NO2 (nitrogen dioxide) level."""
         if self._device.environmental_state:
             return int(self._device.environmental_state.nitrogen_dioxide)
+        return None
 
     @property
     def volatile_organic_compounds(self):
         """Return the VOC (Volatile Organic Compounds) level."""
         if self._device.environmental_state:
             return int(self._device.
                        environmental_state.volatile_organic_compounds)
+        return None
 
     @property
-    def state_attributes(self):
-        """Return the state attributes."""
-        data = {}
+    def unique_id(self):
+        """Return the parent fan's serial number."""
+        return self._device.serial + AIQ_SUFFIX
```

Comment:
Use new style string formatting instead of string concatenation.

Reply:
Done.